internal/xerrors/operation.go

```diff
@@ -101,19 +100,19 @@ func IsOperationError(err error, codes ...Ydb.StatusIds_StatusCode) bool {
 	return false
 }
 
-func (e *operationError) OperationStatus() operation.Status {
+func (e *operationError) Type() Type {
 	switch e.code {
 	case
 		Ydb.StatusIds_ABORTED,
 		Ydb.StatusIds_UNAVAILABLE,
 		Ydb.StatusIds_OVERLOADED,
 		Ydb.StatusIds_BAD_SESSION,
 		Ydb.StatusIds_SESSION_BUSY:
-		return operation.NotFinished
+		return TypeRetryable
 	case Ydb.StatusIds_UNDETERMINED:
-		return operation.Undefined
+		return TypeConditionallyRetryable
 	default:
-		return operation.Finished
+		return TypeNonRetryable
```

Comment:
What about explicit add known unretriable operations as separate case and use default as fallback only.

Reply:
ack